README.md

```diff
@@ -1 +1,173 @@
-# world-cities
\ No newline at end of file
+<a name="readme-top"></a>
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📗 Table of Contents](#-table-of-contents)
+- [📖 Country Details ](#-country-details-)
+  - [🛠 Built With ](#-built-with-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [🚀 Live Demo ](#-live-demo-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites](#prerequisites)
+    - [Setup](#setup)
+    - [Install](#install)
+    - [Usage](#usage)
+    - [Test](#test)
+  - [👥 Authors ](#-authors-)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [⭐️ Show your support ](#️-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝 License ](#-license-)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Country Details <a name="about-project"></a>
+
+**Country Details** is an application where you can fetch countries data in a list format. When user clicks on a country 
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack <a name="tech-stack"></a>
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://legacy.reactjs.org/">React</a></li>
+    <li><a href="https://legacy.reactjs.org/docs/introducing-jsx.html">JSX</a></li>
+    <li><a href="https://www.w3schools.com/css/">CSS</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://restcountries.com/">RestCountries API</a></li>
+  </ul>
+</details>
+
+<details>
+<summary>Database</summary>
+  <ul>
+    <li>none</li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **Countries List**
+- **Country Details**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://country-details-five.vercel.app/)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need: Node.js, NPM on your machine.
+
+### Setup
+
+Clone this repository to your desired folder: by using this command: 
+```bash
+git clone https://github.com/MuhammadUsmanAslam/country-details.git
+```
+
+### Install
+
+Install this project with command: 
+```bash
+npm i
+```
+
+### Usage
+
+To run the project, execute the following command: 
+```bash
+npm run start
+```
+
```

Comment:
- [ ] [REQUIRED] Good job so far! It is not possible to `launch` your project through the `guide` you provided above, please replace it with the correct `launch` guide 🙏🏼 

![image](https://user-images.githubusercontent.com/63915024/233703029-67a06d91-d6f6-44d5-b71b-b1b9fb3940ae.png)

Reply:
It's already working we have to run `npm i` if running first time after cloning

Reply:
@MuhammadUsmanAslam I did implement `npm i` after launching your project that is the result after the commands